client/README.md

```diff
@@ -35,19 +35,21 @@
 
 ### Project Setup
 
-- `npm run setup` to set up the project
-- `npm run reset` to reset the project
+- `npm install` to set up the project
+- `npm run uninstall` to reset the project
 
 ### Build Application
 
-- `npm run package` to build the web application into `dist` folder
+- `npm run pack` to build the web application into `dist` folder
 
 ### Development
 
-- `npm run package` to package the application into `dist` folder
+- `npm run pack` to package the application into `dist` folder
 - `npm run clean` to delete build folders
 - `npm run start` to run the application on Chrome
 - `npm run test` to run unit tests
```

Comment:
npm run test doesn't work


> Caproomster@1.0.0 test /Users/adriannadiaz/PycharmProjects/caproomster/client
> karma start test/test.conf.js

10 03 2017 16:08:54.507:INFO [karma]: Karma v1.5.0 server started at http://0.0.0.0:9876/
10 03 2017 16:08:54.514:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
10 03 2017 16:08:54.523:INFO [launcher]: Starting browser PhantomJS
10 03 2017 16:08:55.537:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket eBA6bbWOEMBuCsN1AAAA with id 26609491
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  TypeError: undefined is not a constructor (evaluating 'angular.module('ngAnimateMock', ['ng'])
    .info({ angularVersion: '1.6.3' })')
  at node_modules/angular-mocks/angular-mocks.js:800
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.078 secs / 0 secs)

npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/Cellar/node/7.6.0/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v7.6.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! Caproomster@1.0.0 test: `karma start test/test.conf.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Caproomster@1.0.0 test script 'karma start test/test.conf.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Caproomster package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     karma start test/test.conf.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Caproomster
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Caproomster
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/adriannadiaz/PycharmProjects/caproomster/client/npm-debug.log